### PR TITLE
add finetune Chinese example

### PR DESCRIPTION
**Pull request by Ariadne330**
- add Chinese version example for Chapter 7 （ based on the released English version )

### CODE REVIEW

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+为了帮助您更好地理解 QLoRA 微调过程，在本教程中，我们提供了一个实用指南，利用 BigDL-LLM 将大语言模型对特定的下游任务进行微调。 这里使用 [Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf) 作为示例来完成文本生成任务。
+
+## 7.1.1 在 Intel GPU 上启用 BigDL-LLM
+
+### 7.1.1.1 安装 BigDL-LLM
+
+按照[Readme](./README.md#70-environment-setup)中的步骤设置环境后，您可以使用以下命令在终端中安装 BigDL-LLM 以及相应的依赖环境：
+
+```bash
+pip install bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```
Comment (plusbang) [inline Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:12]: it maybe better to use `pip install --pre --upgrade`

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+> 您可以使用 Huggingface 库 id 或本地模型路径来指定参数 `pretrained_model_name_or_path`。
+> 如果您已经下载了 Llama 2 (7B) 模型，您可以将 `pretrained_model_name_or_path` 指定为本地模型路径。
+
+通过 BigDL-LLM 优化，您可以使用`bigdl.llm.transformers.AutoModelForCausalLM`替代`transformers.AutoModelForCausalLM`来加载模型来进行隐式量化。
+
+对于英特尔 GPU，您应在`from_pretrained`函数中特别设置 `optimize_model=False`。一旦获得低精度模型，请将其设置为`to('xpu')`。
+
+```python
+model = AutoModelForCausalLM.from_pretrained(pretrained_model_name_or_path = "meta-llama/Llama-2-7b-hf",
+                                            load_in_low_bit="nf4",
```
Comment (plusbang) [inline Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:47]: add space to align

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+>
+> 我们指定 load_in_low_bit="nf4" 以应用 4 位 NormalFloat 优化。 根据 [QLoRA 论文](https://arxiv.org/pdf/2305.14314.pdf)，使用 “nf4” 量化比 “int4” 能达到更优的模型效果。
+
+### 7.1.2.2 准备训练模型
+
+我们可以应用`bigdl.llm.transformers.qlora`中的`prepare_model_for_kbit_training`对模型进行预处理，以进行训练的准备。
+
+```python
+from bigdl.llm.transformers.qlora import prepare_model_for_kbit_training
+model.gradient_checkpointing_enable()
```
Comment (plusbang) [inline Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:63]: It maybe better to comment this line and add comment "# #可以进一步减少内存使用但速度较慢"

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+{'loss': 1.7193, 'learning_rate': 0.0002, 'epoch': 0.03}                                                             
+{'loss': 1.3242, 'learning_rate': 0.00017777777777777779, 'epoch': 0.06}                                             
+{'loss': 1.2266, 'learning_rate': 0.00015555555555555556, 'epoch': 0.1}                                              
+{'loss': 1.1534, 'learning_rate': 0.00013333333333333334, 'epoch': 0.13}                                             
+{'loss': 0.9368, 'learning_rate': 0.00011111111111111112, 'epoch': 0.16}                                             
+{'loss': 0.9321, 'learning_rate': 8.888888888888889e-05, 'epoch': 0.19}                                              
+{'loss': 0.9902, 'learning_rate': 6.666666666666667e-05, 'epoch': 0.22}                                              
+{'loss': 0.8593, 'learning_rate': 4.4444444444444447e-05, 'epoch': 0.26}                                             
+{'loss': 1.0055, 'learning_rate': 2.2222222222222223e-05, 'epoch': 0.29}                                             
+{'loss': 1.0081, 'learning_rate': 0.0, 'epoch': 0.32}
```
Comment (plusbang) [inline Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:158]: The logging information `'learning_rate': 0.0` seems a little strange to me.

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+
+> **注意**
+> `model_path` 参数应该与合并模型的输出路径一致。
+
+我们可以验证微调模型在增添新的数据集训练后是否能做出富有哲理的回答。
+
+```python
+with torch.inference_mode():
+    input_ids = tokenizer.encode('The paradox of time and eternity is', 
+    return_tensors="pt").to('xpu')
```
Comment (plusbang) [inline Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:269]: add indentation

File: Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md (hunk 4 above)
Comment (Ariadne330) [reply Chinese_Version/ch_7_Finetune/7_1_Finetune_Llama2-7B.md:158]:
```shell
GradScaler is enabled, but CUDA is not available.  Disabling.
  warnings.warn("torch.cuda.amp.GradScaler is enabled, but CUDA is not available.  Disabling.")
/opt/anaconda3/envs/yining-qlora/lib/python3.9/site-packages/transformers/optimization.py:411: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning
  warnings.warn(
{'loss': 1.7241, 'learning_rate': 0.0002, 'epoch': 0.03}                                                                    
{'loss': 1.317, 'learning_rate': 0.00017777777777777779, 'epoch': 0.06}                                                     
{'loss': 1.2302, 'learning_rate': 0.00015555555555555556, 'epoch': 0.1}                                                     
{'loss': 1.1478, 'learning_rate': 0.00013333333333333334, 'epoch': 0.13}                                                    
{'loss': 0.937, 'learning_rate': 0.00011111111111111112, 'epoch': 0.16}                                                     
{'loss': 0.9338, 'learning_rate': 8.888888888888889e-05, 'epoch': 0.19}                                                     
{'loss': 0.993, 'learning_rate': 6.666666666666667e-05, 'epoch': 0.22}                                                      
{'loss': 0.86, 'learning_rate': 4.4444444444444447e-05, 'epoch': 0.26}                                                      
{'loss': 1.0082, 'learning_rate': 2.2222222222222223e-05, 'epoch': 0.29}                                                    
{'loss': 1.0047, 'learning_rate': 0.0, 'epoch': 0.32}                                                                       
{'train_runtime': 245.3045, 'train_samples_per_second': 3.261, 'train_steps_per_second': 0.815, 'train_loss': 1.1155864334106445, 'epoch': 0.32}
```
Trained again and got the same result ? ( Didn‘t make change here
